packages/fx-core/tests/component/generator/generator.test.ts

```diff
@@ -337,7 +331,20 @@ describe("Generator error", async () => {
 
   it("fetch sample zip from url error", async () => {
     sandbox.stub(fetchZipFromUrlAction, "run").throws(new Error("test"));
-    const result = await Generator.generateSample(ctx, tmpDir, "bot-sso");
+    const mockedSampleInfo: SampleInfo = {
+      id: "test-id",
+      title: "test-title",
+      shortDescription: "test-sd",
+      fullDescription: "test-fd",
+      tags: [],
+      time: "",
+      configuration: "test-configuration",
+      link: "test-link",
+      suggested: false,
+      url: "https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/test",
+    };
```

Comment:
It seems that you've repeated this object three times within different describe blocks. I think it could be beneficial to identify it as a common mocked object. 

Consider creating a Helper class that owns this type of mocked object and provides some common methods that are useful for your testing toolkit.

Reply:
I think changing this var to be a global var will suffice.